neuralprophet/utils_torch.py

```diff
@@ -49,3 +50,49 @@ def create_optimizer_from_config(optimizer_name, optimizer_args):
     else:
         raise ValueError("The provided optimizer is not supported.")
     return optimizer, optimizer_args
+
+
+def interprete_model(target_model, net, forward_func):
+    """
+    Returns model input attributions for a given network and forward function.
+
+    Parameters
+    ----------
```

Comment:
`target_model` is not defined as parameter in docstring

Reply:
Done